.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Issue Type 
+
+(check one with "x")
+
+```
+[ ] bug report
+[ ] feature request
+```
+
+
+## Current Behavior
+
+
+## Expected Behavior
+
+
+## Steps to Reproduce
+
```

Comment:
And this to steps to reproduce:
* full output of the error you received
* what command you ran
* the code that caused the failure (see [this link](https://help.github.com/articles/basic-writing-and-formatting-syntax/) for help with formatting code)